pkg/provisioner/ironic/ironic.go

```diff
@@ -354,6 +354,44 @@ func (p *ironicProvisioner) findExistingHost() (ironicNode *nodes.Node, err erro
 
 }
 
+func (p *ironicProvisioner) updateAutomatedClean(ironicNode *nodes.Node) (err error) {
+	var updates nodes.UpdateOpts
+	var value bool
+
+	if p.host.Spec.AutomatedCleaningMode == metal3v1alpha1.CleaningModeDisabled {
+		p.log.Info("automated cleaning is disabled for host",
+			"ID", ironicNode.UUID,
+			"automated_clean", ironicNode.AutomatedClean,
+		)
+		p.publisher("CleaningDisabled", "Automated cleaning disabled")
+	} else if p.host.Spec.AutomatedCleaningMode != metal3v1alpha1.CleaningModeDisabled {
```

Comment:
The condition here is redundant, isn't it?

Is there any reason not to always log what cleaning mode is being used? That would let us do something like the below and avoid the conditional entirely.

```go
p.log.Info("setting cleaning mode", 
    "ID", ironicNode.UUID, 
    "mode", p.host.Spec.AutomatedCleaningMode)
value = p.host.Spec.AutomatedCleaningMode != metal3v1alpha1.CleaningModeDisabled

updates = ...
```

Reply:
Yes it is! That's a good point, thanks.